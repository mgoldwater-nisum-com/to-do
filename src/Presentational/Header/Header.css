.title {
  color: rgba(175, 47, 47, 0.15);
  font-weight: 100;
  font-size: 100px;
  line-height: 20px;
}