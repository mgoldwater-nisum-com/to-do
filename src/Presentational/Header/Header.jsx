import React from 'react';
import style from './Header.css';

const Header = () => (
  <h1 className={style.title}>todos</h1>
);

export default Header;