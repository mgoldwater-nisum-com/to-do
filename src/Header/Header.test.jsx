import React from 'react';
import {render, shallow} from 'enzyme';
import Header from './Header';

test('1 + 1 is 2', () => {
  expect(1+1).toBe(2);
});

test('Header component should render', () => {
  expect(shallow(<Header/>).exists()).toBe(true);
});

test('Header component should display the heading todos', () => {
  expect(render(<Header/>).text()).toEqual('todos');
});