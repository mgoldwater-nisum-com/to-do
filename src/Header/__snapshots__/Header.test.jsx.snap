// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `<Header />`;
